### DISCUSSION TITLE
Microsoft.Sysinternals.ProcessExplorer Installer hash does not match.

### DISCUSSION THREAD

**Discussion by Cyberavater** [microsoft/winget-pkgs#101245] [General]
Latest version is 17.03 but winget shows 17.02

Hence, hash mismatch.

**Comment by mdanish-kh** [comment]
Will be resolved once #101247 gets merged. 

For installer mismatch issues / update requests, consider opening issues instead of discussions using the new issue template for betting tracking

![image](https://user-images.githubusercontent.com/88161975/229201076-78a8963d-b4c3-4338-9edf-5ad4c820d7da.png)
